src/neo/SmartContract/Native/NeoToken.cs

```diff
@@ -329,6 +348,8 @@ private async ContractTask<bool> Vote(ApplicationEngine engine, UInt160 account,
             {
                 validator_new.Votes += state_account.Balance;
             }
+            if (gasDistribution is not null)
+                await GAS.Mint(engine, gasDistribution.Account, gasDistribution.Amount, true);
```

Comment:
If there are two votes in the same tx?

Reply:
If two votes come from different account, it doesn't matter. If two votes come from the same account, the second `gasDistribution` must be `null` because there is no gas to distribute.